### PR TITLE
Redesign settings page

### PR DESCRIPTION
**Pull request by jmcgettrick**
- slight redesign of settings page.
- fixed broken tests.
- added confirm javascript message when deleting assignments.

### CODE REVIEW

File: sass/_other.scss [hunk 1]
```diff
@@ -142,50 +142,74 @@ body[class*='-moodle23'] .mod_turnitintooltwo  .centered_cb_cell input {
...
+
+#migrationform #fitem_id_submitbutton, 
+#migration-delete-selected #fitem_id_submitbutton, 
+#fitem_id_enablemigrationtool {
+    width: 50%;
+    margin: 0 auto;
+    background-color: #FFFFFF;
+}
+.fitemtitle {
+    width: 300px !important;
```
Comment (davidmhatton) [inline sass/_other.scss]: Could the `!important` here be rendered un-necessary with greater selector specificity?

File: tests/unit/classes/v1migration/v1migration_test.php [hunk 2]
```diff
@@ -675,10 +675,10 @@ public function test_turnitintooltwo_getassignments() {
         foreach ($assignments as $key => $value) {
             if ($value->migrated == 1) {
                 $checkbox = '<input class="browser_checkbox" type="checkbox" value="'.$value->id.'" name="assignmentids[]" />';
-                $migrationValue = "Yes";
+                $migrationValue = html_writer::tag('i', '', array('class' => 'fa fa-check'));
```
Comment (davidmhatton) [inline tests/unit/classes/v1migration/v1migration_test.php]: c.f. comments above RE accessibility

File: tests/unit/classes/v1migration/v1migration_test.php [hunk 3]
```diff
@@ -675,10 +675,10 @@ public function test_turnitintooltwo_getassignments() {
...
             } else {
                 $checkbox = "";
-                $migrationValue = "No";
+                $migrationValue = html_writer::tag('i', '', array('class' => 'fa fa-times'));
```
Comment (davidmhatton) [inline tests/unit/classes/v1migration/v1migration_test.php]: c.f. comments above RE accessibility

File: tests/unit/classes/v1migration/v1migration_test.php [hunk 4]
```diff
@@ -701,10 +702,10 @@ public function test_turnitintooltwo_getassignments() {
         foreach ($assignments as $key => $value) {
             if ($value->migrated == 1) {
                 $checkbox = '<input class="browser_checkbox" type="checkbox" value="'.$value->id.'" name="assignmentids[]" />';
-                $migrationValue = "Yes";
+                $migrationValue = html_writer::tag('i', '', array('class' => 'fa fa-check'));
```
Comment (davidmhatton) [inline tests/unit/classes/v1migration/v1migration_test.php]: c.f. comments above RE accessibility

File: tests/unit/classes/v1migration/v1migration_test.php [hunk 5]
```diff
@@ -701,10 +702,10 @@ public function test_turnitintooltwo_getassignments() {
...
             } else {
                 $checkbox = "";
-                $migrationValue = "No";
+                $migrationValue = html_writer::tag('i', '', array('class' => 'fa fa-times'));
```
Comment (davidmhatton) [inline tests/unit/classes/v1migration/v1migration_test.php]: c.f. comments above RE accessibility

File: classes/v1migration/v1migration.php [hunk 6]
```diff
@@ -531,10 +531,10 @@ public static function turnitintooltwo_getassignments() {
         foreach ($assignments as $assignment) {
             if ($assignment->migrated == 1) {
                 $checkbox = html_writer::checkbox('assignmentids[]', $assignment->id, false, '', array("class" => "browser_checkbox"));
-                $assignment->migrated = get_string('yes', 'turnitintooltwo');
+                $assignment->migrated = html_writer::tag('i', '', array('class' => 'fa fa-check'));
```
Comment (jmcgettrick) [inline classes/v1migration/v1migration.php]: 👍

**Pull request merged**